.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -41,6 +41,19 @@ body:
       description: Code snippet, link to a similar product, etc.
     validations:
       required: false
+  - type: dropdown
+    id: priority-impact
+    validations:
+      required: true
+    attributes:
+      label: Priority impact
+      multiple: false
+      description: What is the impact to you, your team, or organization? Use discretion and only select "need" priority for high user impact and quality issues. For instance, would someone notice, in a bad way, if this enhancement were present in the release?
```

Comment:
> For instance, would someone notice, in a bad way, if this enhancement were present in the release?

This needs to be rephrased to focus on enhancements not being present in a release.

This applies to `new-component` too.